WooCommerce/Classes/ViewRelated/Products/Edit Product/EditableProductVariationModel.swift

```diff
@@ -133,3 +153,10 @@ extension EditableProductVariationModel: Equatable {
         return lhs.productVariation == rhs.productVariation
     }
 }
+
+extension EditableProductVariationModel {
+    enum Localization {
+        static let anyAttirubteFormat =
```

Comment:
Typo:
```suggestion
        static let anyAttributeFormat =
```

Reply:
Oops, my eyes must have been sore at the time 😓 updated here: https://github.com/woocommerce/woocommerce-ios/pull/2608/commits/796493551517b221555ddf8129a956426a194b26